config/routes.rb

```diff
@@ -1,10 +1,11 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  # Dummy route. This route is never reached in the app, as Omniauth intercepts
-  # it via Rack middleware before it reaches Rails, however adding this route
-  # allows us to use rpi_auth_login_path helpers etc.
+  # Dummy routes. These routes are never reached in the app, as Omniauth
+  # intercepts it via Rack middleware before it reaches Rails, however adding
+  # them allows us to use rpi_auth_login_path helpers etc.
   post '/auth/rpi', as: :rpi_auth_login
```

Comment:
```suggestion
  post '/auth/rpi', as: :rpi_auth_login, params: { login_options: 'v1_signup' }
```

We should be setting this until all apps have migrated off Hydra v0, to ensure that the correct flow is followed post account creation + verification (I know this is the login route but a user could still of course elect to sign up rather than log in).

Reply:
I did wonder about the signup flow not working with v1 logins, but thought it might have been a new issue somehow. 😅